ftag/wps/working_points.py

```diff
@@ -60,6 +60,13 @@ def parse_args(args):
         type=str,
         help='signal flavour ("bjets" or "cjets")',
     )
+    parser.add_argument(
+        "-r",
+        "--rejection",
+        default=None,
+        choices=["ujets", "cjets", "bjets"],
+        help="use rejection instead of efficiency",
```

Comment:
```suggestion
        help="use rejection of specified background class to determine working points",
```

Reply:
I would also change the help of the `-effs` argument to reflect that the values can represent rejections in the case that `--rejection` is set